com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/CustomRenderTexture/CustomRenderTextureSizeNode.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.GraphDelta;
+
+namespace UnityEditor.ShaderGraph.Defs
+{//TODO: define checkings for preview shader
+    internal class CustomRenderTextureSizeNode : IStandardNode
+    {
+        static string Name = "CustomRenderTextureSize";
+        static int Version = 1;
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"
+TextureHeight = CustomRenderTextureHeightRef;
+TextureDepth = CustomRenderTextureDepthRef;
```

Comment:
Here, it's called "CustomRenderTextureDepthRef" but in the params it's called TextureDepthRef.  They need to match.

Reply:
fixed